frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -73,16 +73,17 @@ export const AddCapacityModal = withHandlePromise((props: AddCapacityModalProps)
       path: `/spec/storageDeviceSets/0/count`,
       value: presentCount + 1,
     };
-    props
-      .handlePromise(k8sPatch(OCSServiceModel, ocsConfig, [patch]))
-      .then(() => {
-        setProgress(false);
-        close();
-      })
-      .catch((error) => {
-        setError(error);
-        setProgress(false);
```

Comment:
I think this is still required if something else fails.

Reply:
This is all handled by `withHandlePromise` HOC.

Reply:
You are right, I got confused with other component.
Added back.